src/main/java/application/App.java

```diff
@@ -74,13 +74,21 @@ private static Router createRouter() {
         return app;
     }
 
-    private static int setPortNumber(String[] terminal_args) {
-        return terminal_args.length > 0
-                ? Integer.parseInt(terminal_args[0])
-                : 5000;
-    }
+    private static HashMap<String, String> parseServerSettings(String[] args) {
```

Comment:
Don't forget to update your README

Reply:
I was unable to get the server to run with the command `gradle run -p 1234`. It says that the build failed with an exception, and it seems to think that my port argument was actually a directory argument.
```
* What went wrong:
The specified project directory '/Users/amanda/8L/francesca/java_http_server/1234' does not exist.
```

Reply:
it needs to be run with the following format: ` gradle run --args="-p 3000 -d /test"`

Reply:
You can also do it like this:
```
gradle build
java -jar build/libs/application.jar -p 2000 -d /files
```

Reply:
Ah, ok. I've never used Gradle, but do you need to use the format `--args` and pass it a string? I was hoping it would be used more like my guess of how to use it. No worries if that's the only way, though.

Reply:
I've spent a few hours trying to create a custom gradle task that can handle the args like you mentioned, but it felt like more work than necessary when `--args` can take care of it, so I just ended up using `--args`